project/profiles/capacity4more/modules/c4m/content/c4m_content_photoalbum/c4m_content_photoalbum.module

```diff
@@ -23,6 +23,39 @@ function c4m_content_photoalbum_c4m_og_vocab_info_content() {
 }
 
 /**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Adds an `Add Photos` button to the `Create Photo album` form.
+ */
+function c4m_content_photoalbum_form_photoalbum_node_form_alter(&$form, &$form_instance, $form_id) {
+  $form['actions']['add_photos'] = array(
```

Comment:
I don't think we need to add a new button, just rename the existing one, and add your own `submit_handler` that will do the redirect.